examples/get_imc_data.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+from typing import Mapping, Tuple, Union
+
+import click
+import numpy as np
+from skimage.io import imread
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Coordinates, Features, Indices, Number
+
+
+class ImagingMassCytometryTile(FetchedTile):
+
+    def __init__(self, file_path: str) -> None:
+        """Initialize a TileFetcher for Imaging Mass Cytometry Data"""
+        self.file_path = file_path
+        self._tile_data = imread(self.file_path)
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return self._tile_data.shape
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        # TODO ambrosejcarr: ask about what these coordinates should correspond to.
+        return {
+            Coordinates.X: (0.0, 0.0001),
+            Coordinates.Y: (0.0, 0.0001),
+            Coordinates.Z: (0.0, 0.0001),
+        }
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    def tile_data(self) -> np.ndarray:
+        return self._tile_data
+
+
+class ImagingMassCytometryTileFetcher(TileFetcher):
+    def __init__(self, input_dir: str) -> None:
+        """Implement a TileFetcher for an Imaging Mass Cytometry Experiment.
+
+        This Tile Fetcher constructs spaceTx format from IMC experiments with a specific directory
+        structure:
+
+        input_dir
+        └── <Fov_name>
+            └── <Fov_name>
+                ├── <target_name1>.tiff
+                ├── ...
+                └── <target_nameN>.tiff
+
+        Notes
+        -----
+        - In Imaging Mass Cytometry, each channel specifies a unique target, so channel == target
+        - Imaging Mass Cytometry experiments have only one imaging round, round is hard coded as 1
+        - The spatial organization of the fields of view are not known to the starfish developers,
+          so they are filled by dummy coordinates
+
+        """
+        self.input_dir = input_dir
+
+    @property
+    def _ch_dict(self):
+        channels = [
+            "CD44(Gd160Di)",
+            "CD68(Nd146Di)",
+            "CarbonicAnhydraseIX(Er166Di)",
+            "Creb(La139Di)",
+            "Cytokeratin7(Dy164Di)",
+            "Cytokeratin8-18(Yb174Di)",
+            "E-cadherin(Er167Di)",
+            "EpCAM(Dy161Di)",
+            "Fibronectin(Dy163Di)",
+            "GATA3(Pr141Di)",
+            "Her2(Eu151Di)",
+            "HistoneH3(Yb176Di)",
+            "Ki-67(Er168Di)",
+            "PRAB(Gd158Di)",
+            "S6(Er170Di)",
+            "SMA(Nd148Di)",
+            "Twist(Nd145Di)",
+            "Vimentin(Dy162Di)",
+            "b-catenin(Ho165Di)",
+        ]
+        mapping = dict(zip(range(len(channels)), channels))
```

Comment:
is this just `dict(zip(enumerate(channels)))`?

Reply:
I have written the code you see here like 1000 times and never thought about using enumerate this way. It's actually even simpler!

`dict(enumerate(channels))`